OpenMEEG/libs/OpenMEEG/src/assembleSourceMat.cpp

```diff
@@ -118,27 +118,26 @@ namespace OpenMEEG {
                 domain = geo.domain(domain_name);
             }
             const double sigma = domain.sigma();
-	    //only consider dipoles in non-zero conductivity domain
-	    if(!sigma==0)
```

Comment:
if (sigma!=0.0) ... Ah, OK ,  the hunk was splitted at the wrong place, this is in essence what you did....